lib/base/RequestClient.js

```diff
@@ -56,8 +57,8 @@ RequestClient.prototype.request = function(opts) {
     url: opts.uri,
     method: opts.method,
     headers: opts.headers,
-    httpAgent: opts.forever ? new http.Agent({ keepAlive: true }) : undefined,
```

Comment:
Is `httpAgent` no longer needed?

Reply:
Tbh. Wasn't quite sure. With my testing of it, and by providing proxy or non proxy, I could use the HTTPSAgent.